### PR TITLE
Upgrade hedera-sdk-go to v2.8.0

### PR DESCRIPTION
**Pull request by xin-hedera**
**Description**:

Upgrade hedera-sdk-go to the official v2.8.0 release

**Related issue(s)**:

Fixes #

**Notes for reviewer**:

**Checklist**

- [ ] Documented (Code comments, README, etc.)
- [x] Tested (unit, integration, etc.)

### CODE REVIEW

File: hedera-mirror-rosetta/app/services/construction/crypto_create_transaction_constructor_test.go [hunk 1]
```diff
@@ -150,10 +150,13 @@ func (suite *cryptoCreateTransactionConstructorSuite) TestParse() {
...
-				accountCreateTx, _ := tx.(*hedera.AccountCreateTransaction)
-				accountCreateTx.SetTransactionID(hedera.TransactionID{})
-				return tx
+				return hedera.NewAccountCreateTransaction().
+					SetAccountMemo(memo).
+					SetAutoRenewPeriod(time.Second * time.Duration(autoRenewPeriod)).
+					SetInitialBalance(hedera.HbarFromTinybar(initialBalance)).
+					SetKey(newAccountKey).
+					SetMaxAutomaticTokenAssociations(maxAutomaticTokenAssociations).
+					SetProxyAccountID(proxyAccountId)
```
Comment (xin-hedera) [inline hedera-mirror-rosetta/app/services/construction/crypto_create_transaction_constructor_test.go:159]: in v2.8.0, the SDK made *Transaction.SetTransactionID a one time method, i.e., calling it more than one time will panic:

```
panic: slice is locked [recovered]
	panic: slice is locked
```

File: hedera-mirror-rosetta/app/services/construction/crypto_create_transaction_constructor_test.go [hunk 2]
```diff
@@ -150,10 +150,13 @@ func (suite *cryptoCreateTransactionConstructorSuite) TestParse() {
 		{
 			name: "TransactionIDNotSet",
 			getTransaction: func() interfaces.Transaction {
-				tx := defaultGetTransaction()
-				accountCreateTx, _ := tx.(*hedera.AccountCreateTransaction)
-				accountCreateTx.SetTransactionID(hedera.TransactionID{})
-				return tx
+				return hedera.NewAccountCreateTransaction().
```
Comment (Nana-EC) [inline hedera-mirror-rosetta/app/services/construction/crypto_create_transaction_constructor_test.go:153]: Do you need to specify `MaxTransactionFee` and `TransactionMemo`?

**Review by Nana-EC — COMMENTED** [review]
1 q

**Review by Nana-EC — APPROVED** [review]
LGTM

**Review by steven-sheehy — APPROVED** [review]

**Pull request merged**